tests/modules/basics.c

```diff
@@ -948,6 +956,10 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
         TestGetResp,"readonly",1,1,1) == REDISMODULE_ERR)
         return REDISMODULE_ERR;
 
+    if (RedisModule_CreateCommand(ctx,"test.weird:cmd",
```

Comment:
```suggestion
    /* Add a command with ':' in it's name, so that we can check commandstats sanitization. */
    if (RedisModule_CreateCommand(ctx,"test.weird:cmd",
```